playbooks/devel.yml

```diff
@@ -13,7 +13,4 @@
       - "--disable-system-checks"
       - "--katello-devel-enable-ostree=true"
   roles:
-    - etc_hosts
-    - foreman_server_repositories
-    - foreman_installer_devel_scenario
-    - foreman_installer
+    - katello_devel
```

Comment:
I'd vote for changing this playbooks name to be more explicit to `katello_devel.yml` since it uses the katello_devel role and is focused on Katello and not general development environments.